pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -109,19 +110,29 @@ func (jm *Controller) syncAll() {
 	jobListFunc := func(opts metav1.ListOptions) (runtime.Object, error) {
 		return jm.kubeClient.BatchV1().Jobs(metav1.NamespaceAll).List(opts)
 	}
+
 	jlTmp, err := pager.New(pager.SimplePageFunc(jobListFunc)).List(context.Background(), metav1.ListOptions{})
```

Comment:
Just call `pager.New(...).EachListItem()` instead?

Reply:
And no point building up a slice just to do `for _, sj := range sjs { }` further below. Just replace the for with the `EachListItem()` call.

Reply:
Thx for your comment.
I use the EachListItem to avoid the `for _, sj := range sjs { }`.